apps/omg_watcher/test/omg_watcher/exit_processor/standard_exit_test.exs

```diff
@@ -34,10 +35,15 @@ defmodule OMG.Watcher.ExitProcessor.Core.StandardExitChallengeTest do
 
   @deposit_blknum1 1
   @deposit_blknum2 2
-  @blknum 1_000
+  @early_blknum 1_000
+  @blknum @early_blknum
+  @late_blknum 10_000
+  @blknum2 @late_blknum - 1_000
 
   @utxo_pos_tx Utxo.position(@blknum, 0, 0)
+  @utxo_pos_tx2 Utxo.position(@blknum2, 0, 1)
   @utxo_pos_deposit Utxo.position(@deposit_blknum1, 0, 0)
```

Comment:
naming inconsistency `utxo_pos_deposit` vs `deposit_blknum1`. In the whole file you omit `1`, so maybe it is worth changing to `deposit_blknum`